livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -360,17 +320,17 @@ def _end_speech(self) -> None:
             ],
         )
         self._final_events = []
-        self._event_queue.put_nowait(end_event)
+        self._event_ch.send_nowait(end_event)
         # break async iteration if speech has ended
-        self._event_queue.put_nowait(None)
+        self._event_ch.send_nowait(None)
```

Comment:
We should never send None

Reply:
if the intent is to close, just do nothing, the STT baseclass will close the event_ch when the main task closes